File: CBuilder/DBComparerDemoFIB/IBDataModuleFib.cpp
//---------------------------------------------------------------------------

#include <vcl.h>
#pragma hdrstop

#include "IBDataModuleFib.h"
//---------------------------------------------------------------------------
#pragma package(smart_init)
#pragma link "FIBDatabase"
#pragma link "pFIBDatabase"
#pragma resource "*.dfm"
TDataMdFib *DataMdFib;
//---------------------------------------------------------------------------
__fastcall TDataMdFib::TDataMdFib(TComponent* Owner)
  : TDataModule(Owner)
{
}
//---------------------------------------------------------------------------

File: Delphi/DBComparerDemoFIB/IBDataModuleFib.pas
unit IBDataModuleFib;

interface

uses
  SysUtils, Classes, FIBDatabase, pFIBDatabase;

type
  TDataMdFib = class(TDataModule)
    IBDatabase1: TpFIBDatabase;
    IBDatabase2: TpFIBDatabase;
  private
    { Private declarations }
  public
    { Public declarations }
  end;

var
  DataMdFib: TDataMdFib;

implementation

{$R *.dfm}

end.
